recipes/veloxchem/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex
+
+# clean up unwanted compiler flags
+CXXFLAGS="${CXXFLAGS//-march=nocona}"
```

Comment:
Why do you remove these flags?

Reply:
In local builds these generated broken binaries. But admittedly this was a while back and I'm not sure anymore whether it's necessary anymore. I can try to keep them and see.